.github/workflows/release.yml

```diff
@@ -281,13 +275,20 @@ jobs:
           name: omega-edit-linux.jar
           path: omega-edit-linux.jar
 
+      - name: Download windows jar file
+        uses: actions/download-artifact@v2.1.0
+        with:
+          name: omega-edit-windows.jar
+          path: omega-edit-windows.jar
+
       - name: Move out jar files
         run: |
-          mv omega-edit-mac.jar omega-edit-mac.jar_dir
-          mv omega-edit-linux.jar omega-edit-linux.jar_dir
-          mv omega-edit-mac.jar_dir/omega-edit-mac.jar .
-          mv omega-edit-linux.jar_dir/omega-edit-linux.jar .
-          rm -rf omega-edit-mac.jar_dir omega-edit-linux.jar_dir
+          for file in omega-edit-mac.jar omega-edit-linux.jar omega-edit-windows.jar;
+          do
+            mv $file "${file}_dir"
+            mv "${file}_dir/$file" .
+            rm -rf "${file}_dir"
+          done
```

Comment:
Still curious, what is this accomplishing?

Reply:
See comment above as it works the same way here